### PR TITLE
embedding/csharp: Add StandardMessageCodec

### PR DESCRIPTION
**Pull request by WonyoungChoi**
- #127 
- Add `StandardMessageCodec` and tests.
- This codec will be used by `StandardMethodCodec` later.

### CODE REVIEW

File: embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+                    ["Key_1"] = 1,
+                    ["Key_2"] = "TEST_STRING",
+                    ["Key_3"] = true
+                };
+                byte[] encodedValue = codec.EncodeMessage(content);
+                byte[] expected;
+                using (var stream = new MemoryStream())
+                {
+                    stream.WriteByte(MAP);
+                    CodecTestsHelper.WriteSize(stream, 3);
```
Comment (swift-kim) [inline embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs:215]: It doesn't matter but I just noticed the mixed use of `stream.WriteByte(3)` and `CodecTestsHelper.WriteSize(stream, 3)`.

File: embedding/csharp/Tizen.Flutter.Embedding/Channels/DefaultBinaryMessenger.cs [hunk 2]
```diff
@@ -51,9 +51,6 @@ public static DefaultBinaryMessenger Instance
             }
         }
 
-        /// <summary>
-        /// Sends a binary message to the given channel.
-        /// </summary>
```
[the remark this answers is not in the document]
Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding/Channels/DefaultBinaryMessenger.cs:56]: `DefaultBinaryMessenger` is a internal class. there is no need comments for public documentation.

File: embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs [hunk 3]
```diff
@@ -0,0 +1,360 @@
+﻿// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
```
[the remark this answers is not in the document]
Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs:1]: oops

File: embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs (hunk 1 above)
Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs]: the helper class is needed to simplify code. I added `StandardMessageHelper` class and modified the code more simply.

Comment (swift-kim) [reply embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs]: What I meant was you used `stream.WriteByte(3)` to specify the size of LIST and `WriteSize(stream, 3)` for MAP. Just a small consistency problem.

Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding.Tests/Channels/StandardMessageCodecTests.cs]: Ah I see. I fixed them.

**Review by swift-kim — APPROVED** [review]

**Pull request merged**